bot/exts/info/information.py

```diff
@@ -224,13 +225,22 @@ async def create_user_embed(self, ctx: Context, user: FetchedMember) -> Embed:
             if is_set and (emoji := getattr(constants.Emojis, f"badge_{badge}", None)):
                 badges.append(emoji)
 
+        verified_at, activity = await self.user_verification_and_messages(user)
+
         if on_server:
             joined = time_since(user.joined_at, max_units=3)
             roles = ", ".join(role.mention for role in user.roles[1:])
-            membership = textwrap.dedent(f"""
-                             Joined: {joined}
-                             Roles: {roles or None}
-                         """).strip()
+            if is_mod_channel(ctx.channel):
+                membership = textwrap.dedent(f"""
+                                 Joined: {joined}
+                                 Verified: {verified_at}
+                                 Roles: {roles or None}
+                             """).strip()
+            else:
+                membership = textwrap.dedent(f"""
+                                 Joined: {joined}
+                                 Roles: {roles or None}
+                             """).strip()
```

Comment:
To avoid repetition here, you can try dynamically building this block instead. For example:
```suggestion
            membership = {"Joined": joined, "Verified": verified_at, "Roles": roles or None}
            if not is_mod_channel(ctx.channel):
                membership.pop("Verified")

            membership = textwrap.dedent("\n".join([f"{key}: {value}" for key, value in membership.items()]))
```